### PR TITLE
Allow starting learning rate to be changed in range_test()

### PR DESCRIPTION
**Pull request by davidtvs**
Allows the user to change the starting learning rate when calling `LRFinder.range_test()` instead of relying on what was set at the time the `optimizer` was created.

Also, documents in the README that the `optimizer` shouldn't have a scheduler attached to it.

Related to #11

### CODE REVIEW

**Comment by NaleRaphael** [comment]
I've run this revision and all worked fine!

However, would it be better to throw an exception/warning to remind user about the issue you mentioned? 
> Also, documents in the README that the optimizer shouldn't have a scheduler attached to it.

Beside, it seems would be better to convert `start_lr` to a list and assign the new value to each parameter group of the optimizer. It would be useful if users want to use `LRFinder` with a model in which `lr` is different in some layers (I recall that it called layer-wise learning rate).

File: torch_lr_finder/lr_finder.py [hunk 1]
```diff
@@ -198,6 +205,10 @@ def range_test(
 
         print("Learning rate search finished. See the graph with {finder_name}.plot()")
 
+    def _set_learning_rate(self, new_lr):
```
Comment (NaleRaphael) [inline torch_lr_finder/lr_finder.py]: It would be nice to add some code for conversion and check, e.g.

```python
def _set_learning_rate(self, new_lrs):
    if not isinstance(new_lrs, list):
        new_lrs = [new_lrs]*len(self.optimizer.param_groups)
    if len(new_lrs) != len(self.optimizer.param_groups):
        raise ValueError(
            "Length of `new_lrs` does not equal to the number of `optimizer.param_groups`"
        )

    for i, param_group in enumerate(self.optimizer.param_groups):
        param_group["lr"] = new_lrs[i]
```

A quick example for layer-wise learning rate training scheme:
```python
# file:  pytorch-lr-finder/examples/lrfinder_mnist.ipynb
# change the declaration of optimizer to the following one
optimizer = optim.SGD(
    [
        {'params': model.fc1.parameters(), 'lr': 0.01},
        {'params': model.fc2.parameters(), 'lr': 0.001}
    ],
    lr=0.001, momentum=0.5
)

# then there will be 2 `param_group`s in the optimizer
print(optimizer)
# > SGD (
# Parameter Group 0
#     lr: 0.01
#     ... Omitted below
# Parameter Group 1
#     lr: 0.001
#     ... Omitted below
# )

# if there is only 1 `start_lrs` is given, `lr` in all `param_group` will be set as the same value
lr_finder.range_test(trainloader, end_lr=10, num_iter=100, step_mode="exp", start_lrs=1e-5)

# if multiple `start_lrs` is given, each value in `start_lrs` will be set into each `param_gruop`
lr_finder.range_test(trainloader, end_lr=10, num_iter=100, step_mode="exp", start_lrs=[1e-5, 1e-6])
```

Comment (davidtvs) [reply torch_lr_finder/lr_finder.py]: Good suggestion. Will make the change

**Comment by davidtvs** [comment]
@NaleRaphael think your suggestions are implemented. Thanks for the feedback

**Comment by NaleRaphael** [comment]
That's nice! I've run the latest revision and it worked fine.
Thanks for your effort, too!

**Comment by AmarSaini** [comment]
I'm a bit late to the PR, but everything looks great. Good job lads!
